app/controllers/qsets_controller.rb

```diff
@@ -10,12 +10,14 @@ def index
   # handles the request to show all questions in a qset
   def show
     @feedback_active = !!current_user
-    @questions = Question.includes(:answers).where(qset_id: @qset.id).order(created_at: :desc)
+    # sorts by default by net votes; secondary sort by create date
+    @questions = Question.plusminus_tally.order(created_at: :desc)
```

Comment:
Good comment on line 13. On this line, however, looks like we're missing the `.where(qset_id: @qset.id)` clause, so all questions are showing up now on every qset page.